app/actions/async.js

```diff
@@ -195,7 +195,8 @@ export function doCareLinkUpload(deviceKey, creds, utc) {
           details: err.message,
           utc: actionUtils.getUtc(utc),
           code: 'E_FETCH_CARELINK',
-          version: version
+          version: version,
```

Comment:
Let's leave the Carelink code as is, given it's been deprecated. Part of #719 is to remove the Carelink driver completely, so let's not introduce new merge conflicts.